src/index.js

```diff
@@ -10,20 +10,20 @@ const displayToDoList = () => {
   listContainer.innerHTML = '';
   todoList.forEach((item, index) => {
     const listItem = document.createElement('div');
-    listItem.className = 'list-properties';
+    listItem.className = 'properties-list';
     listItem.innerHTML = `
-    <div class="both-sides">
-      <span class="left-items">
+    <div class="sides">
+      <span class="items-left">
         <input type="checkbox" ${item.completed ? 'checked' : ''}>
         <p>${item.description}</p>
       </span>
-      <span class="right-items">
-        <div id="remove-single"><i class="fa-solid fa-trash"></i></div>
+      <span class=items-right">
+        <div id="single-remove"><i class="fa-solid fa-trash"></i></div>
         <i class="fa-solid fa-ellipsis-vertical"></i>
       </span>
       </div>
     `;
-    listItem.querySelector('#remove-single').addEventListener('click', (event) => {
+    listItem.querySelector('#single-remove').addEventListener('click', (event) => {
```

Comment:
- [ ] Just like the previous reviewer said [here](https://github.com/Jessekay/To-Do-List/pull/2#discussion_r1161125322), You need to add the edit task functionality


- These are the functionalities you're yet to implement.
  - Editing tasks.
  - Toggling completed tasks on and off (true and false or checked and unchecked)

![mobile](https://user-images.githubusercontent.com/41163571/231701004-a6bfbd44-2801-4a71-b938-7eb8c742bf49.gif)